arbitrum-docs/launch-orbit-chain/how-tos/arbos-upgrade.md

```diff
@@ -0,0 +1,49 @@
+
+For example, if your upgrade targets ArbOS 11, you should look for node version `v2.2.2`. The Docker image to use for this version is `offchainlabs/nitro-node:v2.2.2-8f33fea`. Completing this step guarantees that your nodes are prepared and will function smoothly with the ArbOS update.
+
+#### Step 2: Wasm Module Root Upgrade
+
+The `WASM module root` is a 32-byte hash, created from the Merkelized Go replay binary and its dependencies. When ArbOS is upgraded, a new Wasm module root is generated due to modifications in the State Transition Function. This new wasm module root must be set in the rollup contract on the parent chain. To implement the upgrade, you should utilize the [setWasmModuleRoot](https://github.com/OffchainLabs/nitro-contracts/blob/38a70a5e14f8b52478eb5db08e7551a82ced14fe/src/rollup/RollupAdminLogic.sol#L321) method by calling the `Rollup proxy` contract. As the `upgrade executor` contract on the parent chain is the designated owner of the rollup contract, it is necessary for the **chain owner account** to initiate a call to the upgrade executor. This call should include the correct calldata for setting the new wasm module root through the setWasmModuleRoot function within the rollup contract. For example, the wasm module root for ArbOS 11 is `0xf4389b835497a910d7ba3ebfb77aa93da985634f3c052de1290360635be40c4a`. 
+**Note** that wasm module roots are backward compatible, meaning upgrading it prior to an ArbOS version upgrade won't disrupt the system.
+
+#### Step 3: Schedule ArbOS version Upgrade
+
+To update the ArbOS version, you must schedule an upgrade on the Orbit chain. This is done by calling the [scheduleArbOSUpgrade](https://github.com/OffchainLabs/nitro-contracts/blob/acb0ef919cce9f41da531f8dab1b0b31d9860dcb/src/precompiles/ArbOwner.sol#L61) function on the ArbOwner precompile of the Orbit chain. This function requires two inputs:
+1. **newVersion**: Specify the ArbOS version you wish to upgrade to.
+2. **timestamp**: Set the exact timestamp at which you want your Orbit chain to transition to the new ArbOS version.
+
+As with the parent chain, performing **admin actions** on the orbit chain necessitates using the `upgrade executor` contract on the orbit chain. The **chain owner account** must execute a call to the upgrade executor with the appropriate calldata, to invoke the `scheduleArbOSUpgrade` function of the ArbOwner precompile, thereby setting the new ArbOS version and the desired timestamp for the upgrade. 
+**Note** that setting timestamp to `0` causes the upgrade to activate immediately.
+
+By following these three steps, your chain will be successfully upgraded and start using the new ArbOS version after reaching the `timestamp` you specified in step 2. This process ensures a seamless transition to the latest functionalities and improvements of ArbOS.
```

Comment:
```suggestion
Congratulations! You've upgraded your Orbit chain(s) to the specified ArbOS version.
```

One possible way to improve this is to set expectations in the "success" case - something along the lines of "You should see <insert console log or some way to verify that the upgrade has been scheduled successfully>". This way, readers know what to look for if they're not confident that their contract calls were successfully executed.

Not needed, just something to consider as a general pattern (I'm not even sure if this is possible or idiomatic with these contract calls)

Reply:
That's a great suggestion for these type of docs but as you mentioned, for this specific doc, we don't have a success sign, cause it's a complete network upgrade. We can think about it when adding orbit sdk support.
Thanks!